lte/gateway/c/oai/tasks/mme_app/mme_app_ip_imsi.cpp

```diff
@@ -46,18 +48,26 @@ void mme_app_log_ipv4_imsi_map() {
  */
 int mme_app_insert_ue_ipv4_addr(uint32_t ipv4_addr, imsi64_t imsi64) {
   OAILOG_FUNC_IN(LOG_MME_APP);
+  UeIpImsiMap* ueip_imsi_map =
+      magma::lte::MmeNasStateManager::getInstance().get_mme_ueip_imsi_map();
+  if (!ueip_imsi_map) {
+    OAILOG_ERROR_UE(
+        LOG_MME_APP, imsi64,
+        "ueip_imsi_map is NULL while inserting ue_ip:%x \n", ipv4_addr);
+    OAILOG_FUNC_RETURN(LOG_MME_APP, RETURNerror);
+  }
   char ipv4[INET_ADDRSTRLEN] = {0};
   inet_ntop(AF_INET, (void*) &ipv4_addr, ipv4, INET_ADDRSTRLEN);
-  auto itr = ipv4map.find(ipv4);
-  if (itr == ipv4map.end()) {
-    vector<uint64_t> vec = {imsi64};
-    ipv4map[ipv4]        = vec;
+  auto itr_map = ueip_imsi_map->find(ipv4);
+  if (itr_map == ueip_imsi_map->end()) {
+    vector<uint64_t> vec   = {imsi64};
```

Comment:
don't use like this, instead explicitly call std::vector.  Also have `#include <vector>`.

Reply:
Sure I shall remove "using namespace std" and use "std::vector"
#include <vector> is included in file "mme_app_ueip_imsi_map.h", which is included here. So explicitly we don't need to include <vector> in this file.